gato/tasks/control_task.py

```diff
@@ -145,7 +153,7 @@ def evaluate(self, model: GatoPolicy, n_iterations, deterministic=True, promptle
                 # trim to context length
                 input_dict[self.obs_str] = input_dict[self.obs_str][-context_timesteps:,]
                 input_dict[self.action_str] = input_dict[self.action_str][-context_timesteps:,]
-                action = model.predict_control(input_dict, task=self, deterministic=deterministic)
+                action = model.module.predict_control(input_dict, task=self, deterministic=deterministic)
```

Comment:
Having a hard time thinking of a good way to handle this. When the training is launched with `accelerate launch train.py`, then you need to access `model.module.predict...` But when it's launched with `python train.py`, then you need just `model.predict...`. I'd hate to have `if` conditionals all over the place.

Reply:
Oh! Clearly! https://pytorch.org/tutorials/beginner/former_torchies/parallelism_tutorial.html

> Attributes of the wrapped module
> 
> After wrapping a Module with DataParallel, the attributes of the module (e.g. custom methods) became inaccessible. This is because DataParallel defines a few new members, and allowing other attributes might lead to clashes in their names. For those who still want to access the attributes, a workaround is to use a subclass of DataParallel as below.
```
class MyDataParallel([nn.DataParallel](https://pytorch.org/docs/stable/generated/torch.nn.DataParallel.html#torch.nn.DataParallel)):
    def __getattr__(self, name):
        try:
            return super().__getattr__(name)
        except AttributeError:
            return getattr(self.module, name)
```

Reply:
Scratch that. Now I remember. We're getting DataParallel by way of Huggingface's Accelerate library. We'd need to make the change there. Not quite so clear.

Reply:
Oh. Here we go:

```
class GatoPolicy(nn.Module):
    # ...
    @property
    def module(self):
        return self
```

Add that and just use `model.module` everywhere. That ought to work for both Accelerated runs and non-distributed runs.